kuma/users/tests/test_views.py

```diff
@@ -109,6 +110,89 @@ def test_bug_811751_banned_user(self):
         response = self.client.get(url, follow=True)
         self.assertNotEqual(response.status_code, 403)
 
+    def test_get_ban_user_view(self):
+        # For an unbanned user get the ban_user view
+        testuser = self.user_model.objects.get(username='testuser')
+        admin = self.user_model.objects.get(username='admin')
+
+        self.client.login(username='admin', password='testpass')
+        ban_url = reverse('users.ban_user',
+                          kwargs={'user_id': testuser.id})
+
+        resp = self.client.get(ban_url)
+        eq_(200, resp.status_code)
+
+        # For a banned user redirect to user detail page
+        UserBan.objects.create(user=testuser, by=admin,
+                               reason='Banned by unit test.',
+                               is_active=True)
+        resp = self.client.get(ban_url)
+        eq_(302, resp.status_code)
+        ok_(testuser.get_absolute_url() in resp['Location'])
+
+    def test_common_reasons_in_template(self):
+        # The common reasons to ban users (from constance) should be in template
+        testuser = self.user_model.objects.get(username='testuser')
+
+        self.client.login(username='admin', password='testpass')
+        ban_url = reverse('users.ban_user',
+                          kwargs={'user_id': testuser.id})
+
+        resp = self.client.get(ban_url)
+        eq_(200, resp.status_code)
+        page = pq(resp.content)
+
+        reasons_to_ban_found = page.find('a.ban-common-reason')
+        reasons_to_ban_expected = json.loads(
+            constance_config.COMMON_REASONS_TO_BAN_USERS
+        )
+
+        eq_(len(reasons_to_ban_found), len(reasons_to_ban_expected))
+        for reason in reasons_to_ban_found:
+            ok_(reason.text in reasons_to_ban_expected)
+
+        # If there is an error in getting the common reasons from constance,
+        # then 'Spam' should still show up in the template as the default
+        testuser = self.user_model.objects.get(username='testuser')
+
+        self.client.login(username='admin', password='testpass')
+        ban_url = reverse('users.ban_user',
+                          kwargs={'user_id': testuser.id})
+
+        constance_config.COMMON_REASONS_TO_BAN_USERS = 'not valid JSON'
```

Comment:
Instead of setting the config value (which may persist past the end of this test), split into a new test and use the ``@override_config`` decorator, like [other tests](https://github.com/mozilla/kuma/blob/master/kuma/users/tests/test_views.py#L529-L531)